src/content/reference/cli.md

```diff
@@ -265,9 +265,14 @@ $ particle subscribe eventName CoreName
 $ particle subscribe eventName 0123456789ABCDEFGHI
 ```
 
+## particle publish
 
-
-
+  Publishes events to the cloud as if they came from the Particle Core/Photon via Spark.publish (deprecated) or Particle.publish.
```

Comment:
Here's the output: `{"data":"dataData","ttl":"60","published_at":"2015-09-02T00:52:32.331Z","coreid":"001","name":"eventName"}`

<img width="1089" alt="screen shot 2015-09-02 at 08 52 57" src="https://cloud.githubusercontent.com/assets/6760914/9620678/081935d0-5150-11e5-9384-80aafa790406.png">